web-client/src/ducks/auth/facebook/types.ts

```diff
@@ -0,0 +1,14 @@
+import createActionTypeFactory from 'src/store/utils/createActionTypeFactory';
+
+const { asyncType: firebaseAsyncType } = createActionTypeFactory('FIREBASE');
```

Comment:
I was wondering, should this "type factory" be defined as Firebase or as Auth?
Since we are any way implementing a hierarchy, shouldn't this be the name of the module? or at least <Module Name>/<Sub Module or Reducer Name>/type

Reply:
That makes sense!

Reply:
This was corrected